Add logo

jghauser opened the pull request:
I started playing around with adding the logo to the README. I had to use raw html -- `.rst` itself is very inflexible (for instance with centering). Still waiting for the `.svg` because that will look better, so I'll swap that in when I get it. 

Also, where should we add the logos to the repo? At the moment I've put it in root, but that's only for testing. Should I create a new folder (maybe `res`), where we add the various logos (in the various formats)?

I'll also add it to readthedocs (as mentioned by @alexfikl in #40) as well once I get a suitable version of the logo.

I'll probably add a small thank you note at the bottom of the README too once my friend confirms he wants that :).

alejandrogallo commented:
Thank you so much @jghauser for this, I let inkscape trace the png and created an svg

```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- Created with Inkscape (http://www.inkscape.org/) -->

<svg
   version="1.1"
   id="svg1"
   width="617.00067"
   height="616.95319"
   viewBox="0 0 617.00067 616.95319"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs1" />
  <g
     id="g1"
     transform="translate(968.50168,141.88456)">
    <path
       style="fill:#06754b;fill-opacity:1"
       d="m -943.84136,474.72022 c -2.92474,-0.41699 -7.41546,-1.97807 -10.24,-3.55967 -3.30037,-1.84803 -8.61675,-7.16441 -10.46478,-10.46478 -1.66202,-2.96817 -3.14097,-7.35737 -3.59296,-10.66316 -0.48344,-3.53575 -0.48344,-563.29793 0,-566.83369 0.45199,-3.30578 1.93094,-7.69499 3.59296,-10.66315 1.84803,-3.30037 7.16441,-8.61675 10.46478,-10.46479 2.96817,-1.66201 7.35737,-3.14096 10.66316,-3.59296 3.53575,-0.48344 563.29793,-0.48344 566.83368,0 3.30579,0.452 7.69499,1.93095 10.66316,3.59296 3.30037,1.84804 8.61675,7.16442 10.46478,10.46479 1.66202,2.96816 3.14097,7.35737 3.59297,10.66315 0.48344,3.53576 0.48344,563.29794 0,566.83369 -0.452,3.30579 -1.93095,7.69499 -3.59297,10.66316 -1.84803,3.30037 -7.16441,8.61675 -10.46478,10.46478 -2.96817,1.66202 -7.35737,3.14097 -10.66316,3.59297 -3.22903,0.4415 -564.15758,0.40858 -567.25684,-0.0333 z"
       id="path1" />
    <path
       d="m -645.90823,394.2263 c 1.28874,-0.17445 4.43116,-0.4575 6.98316,-0.629 2.552,-0.17151 5.144,-0.46311 5.76,-0.648 0.616,-0.1849 2.128,-0.49375 3.36,-0.68634 2.3198,-0.36263 5.97875,-1.23177 8.48,-2.01431 0.792,-0.24779 2.664,-0.82851 4.16,-1.2905 1.496,-0.46199 3.152,-1.03545 3.68,-1.27437 0.528,-0.23891 1.68,-0.72302 2.56,-1.0758 3.06231,-1.22762 5.76205,-2.39153 6.08,-2.6212 0.176,-0.12713 1.76,-0.98882 3.52,-1.91487 2.86514,-1.50754 11.76093,-7.19244 12.73634,-8.13923 0.21101,-0.20482 0.95966,-0.79588 1.66366,-1.31346 4.54383,-3.34065 12.11643,-10.5881 16.11172,-15.41992 3.09958,-3.74857 8.28604,-10.53018 14.19034,-18.55472 1.53013,-2.07961 3.69013,-4.99118 4.8,-6.47017 1.10987,-1.47898 2.22988,-2.99098 2.48891,-3.36 0.25903,-0.36901 1.63592,-2.25493 3.05976,-4.19093 1.42383,-1.936 3.80389,-5.17443 5.28903,-7.19652 1.48513,-2.02208 3.63624,-4.9272 4.78024,-6.45582 1.144,-1.52863 3.304,-4.45456 4.8,-6.50208 1.496,-2.04752 2.90174,-3.96639 3.12386,-4.26417 1.86335,-2.498 9.77169,-13.20276 10.73868,-14.53592 0.6724,-0.92702 3.3364,-4.54474 5.92,-8.03938 2.5836,-3.49464 6.67327,-9.03864 9.08814,-12.32 2.41488,-3.28136 4.50288,-6.11108 4.64,-6.28828 0.13713,-0.17719 0.96932,-1.30335 1.84932,-2.50258 0.88,-1.19923 2.10891,-2.85426 2.7309,-3.67783 0.622,-0.82358 2.70813,-3.65742 4.63585,-6.29742 1.92773,-2.64 3.61979,-4.944 3.76016,-5.12 0.14036,-0.176 2.05777,-2.768 4.2609,-5.76 6.85871,-9.31456 10.46596,-14.20427 12.29219,-16.66233 0.968,-1.30291 2.31418,-3.12689 2.99151,-4.0533 0.67732,-0.9264 2.88074,-3.91637 4.89647,-6.64437 2.01574,-2.728 4.91756,-6.6628 6.4485,-8.74399 1.53094,-2.0812 3.35952,-4.5571 4.06352,-5.50201 0.704,-0.9449 1.94736,-2.6261 2.76302,-3.736 1.42574,-1.94006 4.03062,-5.47568 15.15698,-20.57263 2.816,-3.82093 7.37525,-9.99325 10.13168,-13.71626 2.75642,-3.72301 5.54197,-7.48911 6.19011,-8.36911 0.64814,-0.88 3.21689,-4.35956 5.70833,-7.73235 2.49143,-3.37279 5.96988,-8.09874 7.72988,-10.50211 1.76,-2.40337 3.86705,-5.25982 4.68233,-6.34765 6.50156,-8.6751 8.52392,-11.66476 11.51177,-17.01789 1.16879,-2.09406 3.14724,-6.149198 3.59084,-7.359994 0.25793,-0.704002 0.91583,-2.502822 1.46201,-3.997372 0.54618,-1.49455 0.99305,-2.9777 0.99305,-3.29589 0,-0.31819 0.2087,-0.9821 0.46377,-1.47535 1.22438,-2.3677 2.72174,-12.21636 2.95554,-19.4397 0.25988,-8.02904 -1.39155,-18.73332 -3.91736,-25.39169 -0.23368,-0.616 -0.51507,-1.48 -0.62532,-1.92 -0.11025,-0.44 -0.85124,-2.2855 -1.64666,-4.10112 -4.1812,-9.54403 -11.29991,-18.60845 -20.15476,-25.66352 -1.43263,-1.14145 -6.37511,-4.7393603 -10.9833,-7.9953601 -4.60818,-3.256 -8.64798,-6.13600006 -8.97733,-6.40000006 -0.32935,-0.26400004 -1.42489,-1.05600004 -2.43453,-1.75999994 -1.00964,-0.704 -3.21369,-2.2518 -4.89788,-3.43955 -1.68419,-1.18776 -3.78217,-2.66677 -4.66217,-3.28669 -0.88,-0.6199299 -2.896,-2.0577199 -4.48,-3.1950999 -1.584,-1.13737 -3.816,-2.72497 -4.96,-3.528 -1.144,-0.80303 -3.16,-2.24387 -4.48,-3.20188 -1.32,-0.958 -3.552,-2.54597 -4.96,-3.52882 -1.408,-0.98285 -3.424,-2.41797 -4.48,-3.18915 -1.056,-0.77119 -2.856,-2.05178 -4,-2.84576 -2.24214,-1.55614 -3.33914,-2.33183 -8.18127,-5.78505 -4.87569,-3.47714 -6.83538,-4.87021 -9.25873,-6.58168 -1.232,-0.87009 -3.032,-2.16022 -4,-2.86697 -0.968,-0.70675 -2.912,-2.08565 -4.32,-3.06423 -1.408,-0.97858 -3.352,-2.35398 -4.32,-3.05645 -0.968,-0.70247 -2.264,-1.6204 -2.88,-2.03985 -0.616,-0.41944 -1.912,-1.34369 -2.88,-2.05388 -5.52967,-4.05695 -7.28265,-5.17056 -9.06538,-5.75892 -0.61492,-0.20293 -1.82519,-0.853237 -2.0959,-0.893273 -0.89051,-0.131698 -2.87239,-0.148358 -4.22342,0.196009 -0.25257,0.06438 -1.8993,0.420244 -2.2953,0.510174 -1.58703,0.36041 -3.38919,0.81852 -5.52,1.40319 -2.03309,0.55785 -6.02157,1.63915 -10.24,2.7761 -0.88,0.23718 -2.752,0.73619 -4.16,1.10892 -1.408,0.37272 -3.568,0.94893 -4.8,1.28047 -1.232,0.33154 -2.888,0.77302 -3.68,0.98106 -0.792,0.20804 -2.376,0.63722 -3.52,0.95372 -1.144,0.31651 -3.232,0.88467 -4.64,1.2626 -1.408,0.37792 -3.50622,0.96512 -4.6627,1.30488 -1.15649,0.33976 -2.30849,0.61566 -2.56,0.61311 -0.25152,-0.003 -1.3933,0.27426 -2.5373,0.61512 -1.144,0.34087 -3.196,0.91895 -4.56,1.28464 -1.364,0.36568 -3.524,0.94866 -4.8,1.29551 -1.276,0.34684 -3.472,0.93165 -4.88,1.29957 -1.408,0.36791 -3.496,0.93147 -4.64,1.25235 -1.144,0.32087 -3.304,0.89225 -4.8,1.26972 -1.496,0.37747 -3.44,0.91801 -4.32,1.2012 -0.88,0.28319 -2.428,0.71101 -3.44,0.95071 -4.69169,1.11127 -5.62376,1.34813 -7.12,1.80934 -0.88,0.27126 -2.824,0.80314 -4.32,1.18195 -1.496,0.37881 -3.656,0.95975 -4.8,1.29097 -1.144,0.33122 -3.232,0.90149 -4.64,1.26726 -1.408,0.36577 -3.604,0.94883 -4.88,1.29567 -1.276,0.34685 -3.4,0.91863 -4.72,1.27063 l -4.8,1.28 -4.8,1.28 c -4.8446,1.29189 -7.4076,1.98293 -9.44,2.54523 -1.144,0.3165 -2.728,0.74194 -3.52,0.94542 -0.792,0.20347 -2.52,0.66376 -3.84,1.02286 -1.32,0.3591 -3.192,0.83896 -4.16,1.06636 -0.968,0.2274 -2.44903,0.64896 -3.29118,0.93679 -0.84215,0.28784 -1.73913,0.52334 -1.99328,0.52334 -0.25416,0 -1.36512,0.27621 -2.46882,0.6138 -1.8501,0.56589 -3.05907,0.89702 -9.36672,2.56545 -3.93713,1.04141 -7.29778,1.94243 -9.6,2.57385 -1.232,0.33789 -3.464,0.92614 -4.96,1.30721 -1.496,0.38108 -3.06466,0.82673 -3.48591,0.99034 -0.42125,0.16361 -1.71725,0.50935 -2.88,0.76831 -1.16275,0.25896 -3.41009,0.83626 -4.99409,1.28287 -1.584,0.44662 -3.79539,1.02142 -4.91419,1.27735 -1.11881,0.25592 -2.18565,0.5589199 -2.37077,0.6733299 -0.18511,0.1144 -1.14173,0.3956 -2.12581,0.62488 -0.98407,0.22928 -3.08523,0.78251 -4.66923,1.22941 -1.584,0.44689 -3.744,1.02106 -4.8,1.27594 -1.056,0.25487 -3.216,0.82937 -4.8,1.27666 -1.584,0.4473 -3.528,0.94857 -4.32,1.11393 -0.792,0.16537 -1.83542,0.45124 -2.31871,0.63526 -0.83547,0.31812 -4.08198,1.18973 -7.92129,2.1266599 -1.63911,0.4 -4.66209,1.22125007 -10.08,2.7384001 -1.232,0.34499 -3.032,0.83084 -4,1.07965 -4.22723,1.08659 -4.75887,1.23208 -11.08968,3.03473 -1.25932,0.35858 -2.48332,0.64988 -2.72,0.64733 -0.23667,-0.003 -1.36632,0.27426 -2.51032,0.61512 -1.144,0.34087 -3.196,0.91895 -4.56,1.28464 -1.364,0.36568 -3.524,0.9486598 -4.8,1.2955098 -1.276,0.34684 -3.472,0.9317503 -4.88,1.2997903 -6.55979,1.71469 -11.77982,3.1374 -14.16672,3.86113 -1.1037,0.33465 -2.19544,0.60845 -2.42611,0.60845 -0.23066,0 -1.34364,0.26905 -2.47328,0.59789 -1.12964,0.32883 -2.77389,0.78164 -3.65389,1.00623 -0.88,0.2246 -2.608,0.68649 -3.84,1.02643 -1.232,0.33994 -3.248,0.88265 -4.48,1.20603 -1.232,0.32338 -2.744,0.73668 -3.36,0.91845 -0.616,0.18177 -2.344,0.64156 -3.84,1.02174 -1.496,0.38018 -3.59415,0.96394 -4.66256,1.29724 -1.06841,0.33329 -2.14841,0.6039 -2.4,0.60135 -0.25159,-0.003 -1.39344,0.27426 -2.53744,0.61512 -2.13504,0.63616 -4.82715,1.37307 -9.6,2.62781 -4.54993,1.19614 -7.72388,3.80042 -9.70331,7.96171 -0.93245,1.96027 -0.99003,2.30103 -1.00024,5.92 -0.0106,3.74261 0.0164,3.89227 1.06175,5.901 0.58993,1.13355 1.88467,2.86265 2.8772,3.84245 1.46081,1.44207 15.30981,12.10406 21.01194,16.17655 0.73929,0.528 11.6741,8.736 24.29958,18.24 12.62548,9.504 23.27477,17.495996 23.66508,17.759996 0.39032,0.264 7.58721,5.664 15.99309,12 8.40589,6.336 15.59881,11.736 15.98428,12 0.38546,0.264 2.68791,1.992 5.11656,3.84 2.42864,1.848 4.7369,3.576 5.12946,3.84 0.39256,0.264 1.63664,1.2 2.76461,2.08 1.12797,0.88 2.37253,1.816 2.76569,2.08 0.39316,0.264 2.40593,1.776 4.47281,3.36 2.06689,1.584 4.36269,3.312 5.10178,3.84 0.7391,0.528 7.05958,5.28 14.04551,10.56 6.98593,5.28 13.0231,9.816 13.41592,10.08 0.39283,0.264 2.70347,1.992 5.13475,3.84 2.43129,1.848 5.025,3.792 5.7638,4.32 0.7388,0.528 3.03435,2.256 5.10124,3.84 2.06688,1.584 4.08118,3.096 4.47621,3.36 0.39503,0.264 2.70747,1.992 5.13875,3.84 2.43129,1.848 5.02411,3.792 5.76182,4.32 0.73772,0.528 8.5324,6.36867 17.32151,12.97927 8.78912,6.6106 20.22821,15.20063 25.42021,19.08896 5.192,3.88832 10.088,7.5922 10.88,8.23084 0.792,0.63864 2.376,1.901 3.52,2.80525 2.64634,2.09174 7.66539,7.16125 10.46853,10.57378 1.19358,1.45304 2.43573,2.9299 2.76034,3.2819 0.83729,0.90793 3.65049,5.67175 6.33052,10.72 1.46085,2.75175 3.73721,8.40003 4.70725,11.68 0.36436,1.232 0.88422,2.98339 1.15525,3.89197 0.27103,0.90859 0.62963,2.56459 0.79689,3.68 0.16727,1.11542 0.45673,2.53198 0.64324,3.14792 0.67615,2.23284 1.05834,11.1906 0.70526,16.52997 -0.51518,7.79084 -1.70021,13.22376 -4.875,22.35014 -2.34823,6.75029 -7.81533,15.87608 -13.15217,21.95384 -5.74538,6.54301 -12.85196,12.30966 -20.42011,16.56993 -1.32,0.74305 -2.616,1.48687 -2.88,1.65293 -1.13587,0.71448 -4.42035,2.2233 -4.83979,2.2233 -0.25179,0 -0.74185,0.1948 -1.08901,0.43288 -0.59516,0.40817 -3.77432,1.44329 -8.4712,2.7582 -3.04271,0.85182 -7.68985,1.81525 -11.52,2.38829 -3.72612,0.55747 -12.50803,0.66778 -14.56,0.18289 -0.704,-0.16636 -2.432,-0.45775 -3.84,-0.64752 -4.06422,-0.54778 -8.96706,-1.53612 -10.13072,-2.0422 -1.316,-0.57234 -3.84076,-1.40287 -5.54928,-1.82547 -2.58666,-0.63981 -11.79718,-5.47724 -16.32,-8.57139 -2.74807,-1.88001 -2.92142,-2.00625 -6.56,-4.77723 -1.32,-1.00526 -2.71167,-2.04165 -3.09261,-2.3031 -0.38093,-0.26144 -7.84424,-5.87535 -16.58512,-12.47535 -8.74088,-6.6 -16.20917,-12.216 -16.59621,-12.48 -0.38703,-0.264 -3.57972,-2.64807 -7.09488,-5.29793 -16.71931,-12.60369 -35.75809,-26.93284 -55.03118,-41.41809 -11.616,-8.73034 -28.68,-21.56862 -37.92,-28.52951 -9.24,-6.96089 -20.76,-15.62286 -25.6,-19.24883 -4.84,-3.62597 -8.944,-6.71254 -9.12,-6.85904 -0.62406,-0.51946 -21.98909,-16.57414 -22.66897,-17.03452 -1.9655,-1.33094 -7.26436,-5.7184 -8.46875,-7.01215 -5.23892,-5.62761 -9.15164,-12.46144 -11.10422,-19.39427 -0.86656,-3.0768 -1.75926,-9.61562 -1.74343,-12.77019 0.0336,-6.69501 2.03926,-14.9748 4.96958,-20.51547 2.95911,-5.5951 6.64112,-10.28776 10.95186,-13.95798 3.47208,-2.95616 5.15402,-4.05861 9.18393,-6.01971 4.13337,-2.01145 11.64541,-4.5008 13.6,-4.50679 0.44,-0.001 1.50689,-0.21689 2.37086,-0.47898 2.05926,-0.6247 7.77614,-0.62274 10.10914,0.003 0.968,0.25982 2.192,0.47366 2.72,0.47519 1.18894,0.003 5.29126,1.12596 7.12856,1.95057 0.75271,0.33783 1.52894,0.61424 1.72495,0.61424 0.19601,0 0.94816,0.27161 1.67144,0.60359 0.72328,0.33197 1.67505,0.76311 2.11505,0.95809 1.28846,0.57097 5.77167,3.50811 7.68,5.03149 0.968,0.77274 4.424,3.38913 7.68,5.81421 6.21827,4.63138 14.4211,10.78011 16.32,12.23326 2.63688,2.01789 9.17686,6.92293 9.81537,7.36161 0.38245,0.26276 2.38866,1.77375 4.45823,3.35775 2.06958,1.584 4.35869,3.312 5.08692,3.84 0.72822,0.528 3.32807,2.472 5.77745,4.32 2.44937,1.848 4.84105,3.648 5.31483,4 0.47378,0.352 2.00372,1.504 3.39986,2.56 1.39614,1.056 2.92803,2.208 3.40419,2.56 0.47616,0.352 1.75616,1.3135 2.84445,2.13666 1.08828,0.82317 3.4187,2.5766 5.1787,3.89653 1.76,1.31993 4.352,3.28148 5.76,4.35902 1.408,1.07753 6.232,4.69919 10.72,8.04812 4.488,3.34893 9.168,6.86166 10.4,7.80608 3.6549,2.80172 12.63098,9.55537 13.38853,10.07359 0.38593,0.264 2.68875,1.992 5.1174,3.84 2.42864,1.848 4.73258,3.576 5.11986,3.84 0.38728,0.264 2.14191,1.56712 3.89918,2.89582 1.75726,1.32869 11.40303,8.58106 21.43503,16.11637 10.032,7.5353 19.19737,14.6378 20.36748,15.78333 2.2975,2.24922 3.50944,4.29864 4.26965,7.22009 1.97443,7.58758 -2.64898,15.95757 -10.16385,18.40014 -2.62809,0.85421 -7.04231,0.94665 -9.03328,0.18917 -0.704,-0.26784 -1.568,-0.57336 -1.92,-0.67893 -0.57293,-0.17182 -4.88191,-2.843 -5.28,-3.27312 -0.088,-0.0951 -1.24,-0.97637 -2.56,-1.95843 -1.32,-0.98205 -4.39101,-3.28605 -6.82446,-5.12 -2.43345,-1.8335 -5.12295,-3.838 -5.97667,-4.454 -0.85372,-0.616 -15.06271,-11.28317 -31.57554,-23.70482 -35.79645,-26.92761 -42.02763,-31.61127 -43.17873,-32.45518 -1.22643,-0.89915 -5.54741,-4.15003 -6.6178,-4.97889 -1.10761,-0.85768 -18.56037,-13.91171 -21.73956,-16.2604 -3.75125,-2.77131 -5.23452,-3.30495 -9.20724,-3.31246 -3.63679,-0.007 -4.9352,0.37856 -7.77783,2.30889 -4.95356,3.3638 -7.29762,10.07317 -5.42323,15.52286 0.96027,2.79192 1.4966,3.7092 3.19797,5.46947 1.29912,1.34408 8.53456,7.01914 13.01797,10.21053 0.74176,0.528 3.03973,2.256 5.10662,3.84 2.06688,1.584 4.07965,3.096 4.47281,3.36 0.39316,0.264 1.63772,1.2 2.76569,2.08 1.12797,0.88 2.37373,1.816 2.76835,2.08 0.65717,0.43964 6.34776,4.75053 7.47165,5.66012 0.7994,0.64698 9.59033,7.25391 10.5645,7.93988 0.49989,0.352 5.18154,3.88 10.40369,7.84 5.22214,3.96 10.47836,7.92 11.68049,8.8 2.23108,1.63322 26.13023,19.6095 26.71132,20.09151 0.176,0.14599 2.336,1.78447 4.8,3.64106 4.41318,3.32527 12.05765,9.10419 14.11111,10.66743 0.57797,0.44 1.74342,1.2945 2.58988,1.89888 0.84645,0.60438 1.61101,1.18514 1.69901,1.29056 0.088,0.10542 0.952,0.75817 1.92,1.45056 0.968,0.69239 1.832,1.34224 1.92,1.44413 0.34372,0.39797 6.62282,4.36623 7.68,4.85362 2.04758,0.94397 4.2968,1.81181 6.4,2.46939 6.60015,2.06359 5.88695,1.95286 12.57856,1.95286 5.61699,0 6.61267,-0.0829 9.26345,-0.77167 6.86245,-1.78302 10.59679,-3.38637 15.17229,-6.51428 5.33722,-3.64863 10.11319,-8.49502 12.7457,-12.93359 1.41069,-2.37852 2.37487,-4.20115 3.01557,-5.70046 0.41365,-0.968 0.94663,-2.192 1.18439,-2.72 0.81745,-1.81531 1.98367,-7.04553 2.3762,-10.65674 0.57965,-5.33256 0.0619,-11.08623 -1.45594,-16.17971 -1.68294,-5.64753 -2.56976,-7.50793 -6.50368,-13.64355 -1.59626,-2.48964 -6.93435,-7.6513 -11.45543,-11.07682 -2.11733,-1.60425 -5.90846,-4.48425 -8.42473,-6.4 -2.51627,-1.91575 -4.78694,-3.62718 -5.04593,-3.80318 -0.51674,-0.35116 -51.2532,-38.37323 -51.77045,-38.79694 -0.176,-0.14418 -4.712,-3.55783 -10.08,-7.58591 -5.368,-4.02807 -10.192,-7.65323 -10.72,-8.05591 -0.528,-0.40268 -10.536,-7.90994 -22.24,-16.6828 -11.704,-8.77286 -28.81728,-21.63141 -38.02952,-28.57455 -15.67699,-11.81553 -20.03436,-15.02389 -20.40445,-15.02389 -0.0836,0 -0.54211,-0.2686 -1.01899,-0.59689 -2.42119,-1.666814 -4.80764,-3.089506 -7.03587,-4.194486 -2.19231,-1.08716 -3.64554,-1.68476 -7.75117,-3.18747 -1.71928,-0.62927 -6.39055,-1.93975 -8.002,-2.24488 -0.79311,-0.15017 -2.23311,-0.4894 -3.2,-0.75384 -3.46224,-0.94688 -11.66855,-1.33835 -18.87309,-0.9003 -8.7512,0.53209 -15.235,1.96103 -21.28491,4.6909 -0.528,0.23825 -1.824,0.81291 -2.88,1.27703 -3.54288,1.55712 -7.34844,3.501502 -8.74818,4.469716 -0.7635,0.52812 -1.6275,1.10597 -1.92,1.2841 -4.40527,2.68286 -12.80952,10.01873 -15.41182,13.45261 -0.264,0.34836 -1.06889,1.35116 -1.78865,2.22845 -5.54756,6.76169 -10.84765,16.86208 -13.09494,24.95506 -1.80735,6.50863 -2.35383,9.26995 -2.75486,13.92 -0.37786,4.38147 -0.1876,14.85217 0.32853,18.08 0.74635,4.66757 1.96726,9.70221 3.06524,12.64 1.98623,5.31445 2.62368,6.86879 3.67459,8.96 6.23181,12.40068 13.00559,20.28363 25.43515,29.6 1.87851,1.408 4.56192,3.424 5.96315,4.48 1.40123,1.056 3.66012,2.75515 5.01974,3.77588 1.35963,1.02074 4.48805,3.37587 6.95205,5.23362 2.464,1.85775 4.80754,3.6146 5.20787,3.90411 0.40033,0.28952 1.87309,1.39039 3.2728,2.44639 1.39972,1.056 3.73618,2.81258 5.19214,3.90351 7.6085,5.70097 18.88932,14.18322 19.76719,14.8633 0.44,0.34086 3.248,2.45683 6.24,4.70215 2.992,2.24532 13.864,10.42081 24.16,18.16775 10.296,7.74695 29.16799,21.92389 41.93776,31.50432 12.76976,9.58043 23.59755,17.70697 24.06175,18.05897 0.46421,0.352 2.82703,2.152 5.25071,4 2.42369,1.848 4.7213,3.576 5.1058,3.84 0.3845,0.264 4.13369,3.08148 8.33154,6.26107 4.19784,3.17958 9.50444,7.18059 11.79244,8.89112 7.44261,5.56417 8.18136,6.11884 9.05645,6.79967 1.61675,1.25787 9.28954,7.00627 9.99495,7.48814 0.38647,0.264 1.01,0.71003 1.38563,0.99118 5.32113,3.98271 15.45163,9.62992 21.16297,11.79722 1.056,0.40072 2.568,0.98194 3.36,1.2916 2.37397,0.92818 4.82105,1.77677 7.36,2.55229 1.32,0.40319 2.76,0.87385 3.2,1.04591 0.44,0.17205 1.808,0.496 3.04,0.71988 1.232,0.22389 3.464,0.63886 4.96,0.92218 6.66341,1.26192 16.73788,1.80676 21.49684,1.16256 z m 99.35649,-117.97615 c -0.11733,-0.11734 -0.21124,-0.44134 -0.20869,-0.72 0.003,-0.27867 -0.29551,-1.51467 -0.66234,-2.74667 -0.36683,-1.232 -0.93663,-3.176 -1.26621,-4.32 -1.39866,-4.85481 -2.03207,-6.94586 -2.42332,-8 -0.22863,-0.616 -0.50749,-1.48 -0.61968,-1.92 -0.59343,-2.32725 -2.97492,-8.13781 -5.31173,-12.96 -5.0403,-10.40109 -11.13566,-18.98699 -19.32136,-27.216 -4.00568,-4.02687 -10.74853,-9.99399 -12.79888,-11.32641 -0.42113,-0.27368 -4.84166,-3.58336 -9.82341,-7.35485 -16.71573,-12.65487 -17.99694,-13.62089 -18.70415,-14.10274 -0.59044,-0.40229 -12.89295,-9.67013 -20.27356,-15.27267 -4.76819,-3.61948 -10.98795,-8.27916 -11.63042,-8.71321 -0.37927,-0.25623 -1.61552,-1.19223 -2.74722,-2.08 -1.1317,-0.88777 -2.3844,-1.83012 -2.78376,-2.09412 -0.64497,-0.42635 -8.45103,-6.27704 -12.2786,-9.20288 -0.792,-0.60541 -5.688,-4.28763 -10.88,-8.1827 -5.192,-3.89507 -9.584,-7.20256 -9.76,-7.34999 -0.176,-0.14743 -3.056,-2.31646 -6.4,-4.82008 -3.344,-2.50362 -10.45374,-7.8578 -15.79941,-11.89819 -5.34568,-4.04039 -10.03376,-7.56216 -10.41797,-7.82616 -0.38421,-0.264 -2.6843,-1.992 -5.11132,-3.84 -2.42702,-1.848 -4.82138,-3.648 -5.32081,-4 -0.49943,-0.351998 -9.7376,-7.287546 -20.52927,-15.412326 -15.86928,-11.94759 -39.8683,-29.98799 -44.55286,-33.49101 -0.51241,-0.38316 -1.97225,-1.48866 -3.24411,-2.45666 -1.27185,-0.968 -2.5703,-1.91319 -2.88545,-2.10043 -0.98983,-0.58808 -0.29234,-1.29378 1.83181,-1.85335 1.11616,-0.29403 3.03739,-0.81181 4.26939,-1.15062 1.232,-0.3388 3.392,-0.91713 4.8,-1.28518 2.89637,-0.75709 6.59077,-1.75687 8.96,-2.42475 0.88,-0.24807 2.464,-0.67532 3.52,-0.94945 1.056,-0.27413 2.928,-0.78181 4.16,-1.12817 1.232,-0.34637 3.104,-0.83967 4.16,-1.09623 1.96528,-0.47747 4.52122,-1.15574 7.36,-1.95311 0.88,-0.24718 2.968,-0.8136 4.64,-1.25871 1.672,-0.44511 3.76,-1.01991 4.64,-1.27733 1.75804,-0.51426 4.48103,-1.2346 7.2,-1.90469 1.695,-0.41772 3.91228,-1.00977 9.2,-2.45653 1.364,-0.3732 3.272,-0.88049 4.24,-1.12731 0.968,-0.24683 2.48,-0.65939 3.36,-0.91681 1.60589,-0.46976 6.1535,-1.70116 9.44,-2.55616 1.67086,-0.43468 3.22031,-0.84517 9.12,-2.41607 1.408,-0.37491 3.496,-0.94307 4.64,-1.26258 4.72309,-1.31914 5.98617,-1.65397 7.30076,-1.93537 0.75941,-0.16257 2.05541,-0.51561 2.88,-0.78455 0.82458,-0.26893 2.36324,-0.70768 3.41924,-0.975 4.64167,-1.17502 5.52102,-1.40032 6.88,-1.76281 1.79981,-0.48007 5.27679,-1.44227 8,-2.21386 1.144,-0.32415 3.304,-0.89842 4.8,-1.27618 1.496,-0.37775 3.656,-0.9509403 4.8,-1.2737603 1.144,-0.32282 3.304,-0.90243 4.8,-1.2880198 1.496,-0.38559 3.59415,-0.97825 4.66256,-1.31702 1.06841,-0.33877 2.16015,-0.61595 2.42611,-0.61595 0.26595,0 1.4078,-0.27073 2.53744,-0.60163 1.12964,-0.33089 3.20589,-0.9169 4.61389,-1.30225 1.408,-0.38534 3.50622,-0.97861 4.6627,-1.31837 1.15649,-0.33976 2.30849,-0.61566 2.56,-0.61311 0.25152,0.003 1.3933,-0.27426 2.5373,-0.61512 1.144,-0.34087 3.232,-0.9273 4.64,-1.30317999 1.408,-0.37588004 3.496,-0.94922004 4.64,-1.27409007 1.144,-0.32487 3.304,-0.89985004 4.8,-1.27774004 3.03649,-0.7670099 6.48914,-1.7092899 9.57389,-2.6128799 1.12964,-0.3309 2.27149,-0.60163 2.53744,-0.60163 0.26596,0 1.3577,-0.27557 2.42611,-0.61239 1.06841,-0.33681 3.09456,-0.91703 4.50256,-1.28937 1.408,-0.37234 3.604,-0.96077 4.88,-1.30761 3.25408,-0.88453 6.52972,-1.7585699 9.52,-2.5402099 1.408,-0.36804 3.604,-0.95295 4.88,-1.29979 1.276,-0.34685 3.436,-0.92983 4.8,-1.29551 1.364,-0.36569 3.416,-0.94377 4.56,-1.28464 1.144,-0.34086 2.32909,-0.61767 2.63352,-0.61512 0.30444,0.003 1.09644,-0.19387 1.76,-0.43649 0.66357,-0.24262 2.71848,-0.82916 4.56648,-1.30342 1.848,-0.47425 4.296,-1.14183 5.44,-1.4835 1.144,-0.34167 2.25698,-0.61914 2.47328,-0.61659 0.2163,0.003 1.2963,-0.27157 2.4,-0.60916 1.1037,-0.33759 3.15872,-0.91768 4.56672,-1.2891 1.408,-0.37142 3.496,-0.94136 4.64,-1.26655 2.4375,-0.69286 4.28784,-1.18117 6.72,-1.77342 1.64838,-0.4014 5.16798,-1.3439 10.4,-2.78498 1.232,-0.33933 3.032,-0.8218 4,-1.07215 0.968,-0.25035 2.768,-0.73463 4,-1.07617 4.34658,-1.205 8.6478,-2.36555 9.30442,-2.51051 0.63663,-0.14054 1.51554,0.36885 4.77558,2.76781 2.05213,1.51009 8.76285,6.31545 11.2,8.01999 1.232,0.86166 2.90465,2.04191 3.71701,2.62277 0.81235,0.58086 3.10162,2.20811 5.08726,3.61611 1.98563,1.408 4.34498,3.09048 5.24299,3.73885 0.89801,0.64837 2.78474,1.9906299 4.19274,2.9828099 2.24059,1.57888 5.09837,3.61402 8.32,5.9250099 0.528,0.37875 2.76,1.98004007 4.96,3.5584201 2.2,1.57837 4.35991,3.11393 4.7998,3.41234 1.04518,0.70903 4.46359,3.13471 9.6726,6.8636101 1.53583,1.09943 4.02355,2.86296 5.52827,3.91896 1.50473,1.056 3.50622,2.496 4.44776,3.2 0.94155,0.704 2.96476,2.144 4.49604,3.2 1.53128,1.056 3.55456,2.496 4.49616,3.2 1.76012,1.31596 4.19828,3.03971 10.87937,7.69157 5.47349,3.81105 7.32041,5.3518 8.87373,7.40277 0.75145,0.99218 1.82756,2.41867 2.39135,3.16997 1.22732,1.6355 3.55215,6.01219 4.49154,8.45569 1.93091,5.02264 2.96395,10.74425 2.96363,16.41446 -6.8e-4,11.97827 -3.39812,21.57626 -11.52027,32.545536 -5.41614,7.3147 -6.03176,8.14404 -7.83871,10.56 -1.76567,2.36078 -8.47093,11.44959 -11.68595,15.84 -0.90217,1.232 -3.13394,4.256 -4.95949,6.72 -1.82556,2.464 -4.0597,5.488 -4.96477,6.72 -0.90506,1.232 -3.46931,4.70303 -5.69832,7.71341 -2.22901,3.01038 -4.91674,6.6589 -5.97274,8.10784 -1.056,1.44893 -2.712,3.68862 -3.68,4.97708 -0.968,1.28846 -3.848,5.17796 -6.4,8.64334 -2.552,3.46538 -6.41084,8.69866 -8.57519,11.62951 -2.16436,2.93085 -4.97236,6.7364 -6.24,8.45678 -1.26765,1.72037 -3.24081,4.38637 -4.38481,5.92443 -1.144,1.53806 -3.16,4.27577 -4.48,6.0838 -1.32,1.80803 -3.12,4.2583 -4,5.44504 -3.66654,4.94461 -11.12147,15.03056 -16.95497,22.93877 -1.42809,1.936 -3.88544,5.26649 -5.46078,7.40109 -1.57534,2.1346 -5.16825,7.00439 -7.98425,10.82175 -2.816,3.81736 -6.056,8.19316 -7.2,9.72401 -3.27509,4.38257 -7.4739,10.0663 -8,10.82921 -2.33251,3.38246 -5.0031,6.82394 -5.29538,6.82394 -0.19687,0 -0.45395,-0.096 -0.57129,-0.21333 z m -44.04883,-120.60656 c 3.88419,-0.73682 7.01434,-2.37808 9.74589,-5.11015 3.38123,-3.38188 3.93803,-4.75855 4.49858,-11.12255 0.25131,-2.85308 0.67939,-7.70741 0.95129,-10.78741 0.27191,-3.08 0.70784,-8.192 0.96875,-11.36 0.2609,-3.168 0.54011,-6.552 0.62047,-7.52 0.48202,-5.80631 1.46029,-18.506576 1.76239,-22.879996 0.19452,-2.816 0.49565,-6.94379 0.66917,-9.17286 0.2761,-3.54674 0.24768,-4.22608 -0.22759,-5.44 -2.1654,-5.53076 -7.68742,-8.75857 -17.39345,-10.16707 -1.32,-0.19155 -4.776,-0.55743 -7.68,-0.81307 -4.81302,-0.42368 -16.64404,-1.61348 -23.84,-2.39749 -1.584,-0.17258 -4.968,-0.54171 -7.52,-0.82029 -7.69787,-0.84031 -19.75389,-2.03721 -38.56,-3.82816 -4.576,-0.43578 -9.904,-0.94673 -11.84,-1.13544 -20.67809,-2.01561 -20.60305,-2.01246 -26.07107,-1.09687 -9.71444,1.62661 -15.10737,5.24788 -15.15372,10.17548 -0.0339,3.60379 2.49817,6.43052 8.34622,9.31749 5.93656,2.93066 9.26701,3.36185 40.87857,5.29253 3.168,0.19349 12.672,0.78191 21.12,1.30761 8.448,0.52571 18.816,1.16968 23.04,1.43106 4.224,0.26138 10.76283,0.6894 14.53073,0.95116 3.7679,0.26176 7.28041,0.47592 7.80559,0.47592 h 0.95485 l -0.19713,1.88605 c -0.10842,1.03733 -0.78831,5.53733 -1.51085,10 -0.72255,4.46267 -1.57757,9.841946 -1.90004,11.953946 -0.98351,6.44146 -3.14319,20.01131 -5.41527,34.02566 l -0.6397,3.94566 0.65929,1.76209 c 1.1644,3.11216 2.59306,5.33784 4.4425,6.92089 4.90058,4.19471 10.14933,5.49674 16.95453,4.20581 z m 92.7035,-39.84866 c 1.71405,-0.61875 2.50747,-1.33022 10.652,-9.55194 15.55022,-15.697576 40.50276,-41.103316 41.54715,-42.301746 1.41124,-1.6194 1.42676,-3.86555 0.0434,-6.27776 -0.5299,-0.924 -1.07119,-1.68 -1.20286,-1.68 -0.13166,0 -0.94443,-0.58261 -1.80614,-1.29469 -3.05878,-2.52762 -7.8353,-3.89716 -9.85669,-2.82613 -0.45758,0.24245 -3.02185,2.31282 -5.69839,4.60082 -2.67653,2.288 -7.62042,6.50641 -10.98643,9.37425 -3.366,2.86784 -7.27201,6.19997 -8.68001,7.40472 -1.408,1.20476 -5.224,4.45233 -8.48,7.21682 -3.256,2.76449 -6.856,5.84599 -8,6.84778 -2.02408,1.77246 -5.24762,4.49396 -10.20816,8.61836 -4.49885,3.740518 -5.77285,5.656196 -5.78456,8.698066 -0.0177,4.5858 4.80567,9.80016 10.5245,11.37775 2.35948,0.65089 5.81014,0.56119 7.93622,-0.2063 z"
       style="fill:#e6fe6a;fill-opacity:1"
       id="path1-3" />
  </g>
</svg>
```

This should give a quite good version of the logo until your friend gives us the original SVG file?

alejandrogallo commented:
here the file, maybe it's easier to download like that
![logo](https://github.com/papis/papis/assets/12715358/19930e83-d892-4a6c-a32e-cfd62118189d)

jghauser commented:
Thanks @alejandrogallo! My friend is gonna do all the logos we need today, so we should have the original `.svg`s (also for the various variations we need + the github social preview).

jghauser commented:
@alejandrogallo, actualllyyyy my friend isn't much of a vector graphics person, so converting these to `svg`s is after all necessary. can you tell me what settings you used to get such nice results in tracing the png?

jghauser commented:
> What format and/or application did they use to create the logos and play around with them? Maybe we can convert from those?

Photoshop. He did manage to export svgs, but they just contain embedded pngs as far as i can tell.

alexfikl commented:
> Photoshop. He did manage to export svgs, but they just contain embedded pngs as far as i can tell.

Hm, maybe he can import them into Illustrator instead and export from there? That should be more vector graphics-friendly. Otherwise not sure.. haven't used those things since high school :(

Anyways! The traced PNG by @alejandrogallo looks very smooth, so that should work just fine, no need to worry about it :grin:

alejandrogallo commented:
Ok great! I just did the trace bitmap function in inkscape, I love inkscape btw.
The settings I don't know it worked quite fine out of the box, I just fixed one corner by hand and that was all.

Thank you all!

jghauser commented:
This is done I think! I added three things to a new `res` folder:
1. logo
2. white logo + "Papis" (for docs)
3. github social preview (@alejandrogallo will need to add this as described [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview). It might have to be converted to png first. If I understand things correctly github will extract the logo from this and show it at the top left of our page.

I've added the logo to our docs. (I also fixed a broken link in the quick guide that I noticed while working on this in a second commit in this PR.)

I've also added a short note to the README, thanking my friend for creating the logo :).

alexfikl commented on README.rst:
```diff
@@ -1,9 +1,29 @@
+   <div align="center">
 
-Papis is a powerful and highly extensible CLI document and bibliography manager.
+   <img src="res/logo.svg" width=300>
```
Maybe rename the folder to `resources`, so that it's clearer?

Otherwise, yeah, can't think of a better location for the logos.

jghauser replied:
Ok, done!

alexfikl reviewed (approved):
This looks great! I didn't even know about that Github social thing :grin: I left a comment, but otherwise feel free to merge this when you think it's ready! :rocket: 

What fonts did you use in the social preview?

jghauser commented:
The fonts are
- *Poppins* for the text "Papis"
- *Hack* for the description in the github social preview

Both fonts have open licenses.

jghauser commented:
Thanks everyone for all the help with this. We now have a beautiful logo! :heart: :tada:

jghauser commented:
Except if @alejandrogallo was super quick, I seem to have misunderstood how the logo gets added. It's already there, I assume github somehow took it from the README? :shrug: :tada:

alexfikl commented:
> Except if @alejandrogallo was super quick, I seem to have misunderstood how the logo gets added. It's already there, I assume github somehow took it from the README? 🤷 🎉

I think you can also just add it like a normal profile picture for the organization. That Github social thing seems to be more for showing some customized thumbnail when linking the repo? Unless I misunderstood how that works too :))
